src/components/list/list.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { ListItem } from '../listItems/listItem';
+import './list.scss';
+
+interface Prop {
+  obj: Array<Array<number | string>>;
+  width?: string | number | undefined;
+  height?: string | number | undefined;
+}
+
+export const List = ({ obj, width, height }: Prop) => (
```

Comment:
Component files always need to be Capitalize and match scss file 
List.tsx
List.scss

Reply:
src/components/list/List.tsx for example

Reply:
👍